sapi/fuzzer/generate_parser_corpus.php

```diff
@@ -12,7 +12,7 @@
 foreach ($it as $file) {
     if (!preg_match('/\.phpt$/', $file)) continue;
     $code = file_get_contents($file);
-    if (!preg_match('/--FILE--(.*)--EXPECT/s', $code, $matches)) continue;
+    if (!preg_match('/--FILE--(.*?)--(?:CLEAN|EXPECT|STDIN|XFAIL)/s', $code, $matches)) continue;
```

Comment:
`--FILE--(.*?)\R--([_A-Z]+)--` maybe?

Reply:
It works too.
Updated.

Reply:
Even better:
`--FILE--\R(.*?)\R--([_A-Z]+)--`